src/pages/Build.vue

```diff
@@ -220,6 +228,15 @@ export default defineComponent({
         return {label: distribution.name, value: distribution.name}
       })
     },
+    failedItems () {
+      let rebuilt = true
+      for (let task of this.build.tasks) {
+        if (task.status !== BuildStatus.COMPLETED) {
```

Comment:
why COMPLETED ? If we need failed to rebuild

Reply:
My bad, fixed